openssh/files/default/ssh_known_hosts

```diff
@@ -81,26 +81,19 @@
 {%- set host_names = salt['mine.get'](target, hostname_function, tgt_type=tgt_type) -%}
 
 {#- Salt SSH (if any) #}
-{%- for minion_id, minion_host_keys in salt['pillar.get'](
-  'openssh:known_hosts:salt_ssh:public_ssh_host_keys',
-  {}
-).items() -%}
+{%- for minion_id, minion_host_keys in known_hosts | traverse('salt_ssh:public_ssh_host_keys', {}) | dictsort -%}
 {%-   if salt["match.{}".format(tgt_type)](target, minion_id=minion_id) -%}
 {%      do host_keys.update({minion_id: minion_host_keys}) %}
 {%-   endif -%}
 {%- endfor -%}
-{%- for minion_id, minion_host_names in salt['pillar.get'](
-  'openssh:known_hosts:salt_ssh:public_ssh_host_names',
-  {}
-).items() -%}
+{%- for minion_id, minion_host_names in known_hosts | traverse('salt_ssh:public_ssh_host_names', {}) | dictsort -%}
```

Comment:
Likewise here about the use of `| dictsort`, as mentioned in the last comment.

Reply:
Ok, same as the above.